.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -0,0 +1,47 @@
+---
+name: 🐞 Bug Report
+about: Something is broken? 🔨
+---
+
+### Bug Report
+
+<!-- Fill in the relevant information below to help triage your issue. -->
+
+|    Q               |   A
+|--------------------| ------
+| Breaking change    | yes/no
+| Version            | x.y.z
```

Comment:
I saw @PascalPrecht talking about a hint where they can find this version? Should we add something like this in the comment above?

> Fill in the relevant information below to help triage your issue. You can find the client version when inspecting the body or you can find it in the relevant package.json file.